Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -1217,61 +1202,46 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
-		FFC8CDA52AA3867A00D129A6 /* XCRemoteSwiftPackageReference "firebase-ios-sdk" */ = {
+		8028219F2CAD786200548B6A /* XCRemoteSwiftPackageReference "firebase-ios-sdk" */ = {
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/firebase/firebase-ios-sdk.git";
 			requirement = {
 				kind = upToNextMajorVersion;
-				minimumVersion = 10.14.0;
+				minimumVersion = 11.3.0;
```

Comment:
also don't update packages, this is something that should explicitly be in it's own PR, not mixed with other things

again, make a new issue / PR for this (but just make an issue, this is something that someone else can do as well to contribute to the project